### PR TITLE
Add an example of adding/removing a tiering policy for caggs

### PR DESCRIPTION
**Pull request by zilder**
# Description

We extended the `add_tiering_policy()` / `remove_tiering_policy()` functions onto continuous aggregate

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by iroussos — APPROVED** [review]
Looks great @zilder. approving on my side

**Comment by iroussos** [comment]
All yours @billy-the-fish to merge or send it back if you don't like something

This is a small update that introduces our newest ability for Tiered Storage: To directly tier continuous aggregates using the standard interface we use for hypertables as well.

File: use-timescale/data-tiering/creating-data-tiering-policy.md [hunk 1]
```diff
@@ -15,16 +15,16 @@ older than the `move_after` threshold are moved. This works similarly to a
...
 
 To add a tiering policy, use the `add_tiering_policy` function:
 
 ```sql
-SELECT add_tiering_policy(hypertable REGCLASS, move_after INTERVAL);
+SELECT add_tiering_policy(hypertable REGCLASS, move_after INTERVAL, if_not_exists BOOL = false);
 ```
 
-In this example, you use a hypertable called example, and tier chunks older than three days.
+Tiering policy can be added for either a hypertable or a continuous aggregate. In this example, you use a hypertable called `example`, and tier chunks older than three days.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/creating-data-tiering-policy.md]:
```suggestion
You can add a tiering policy to hypertables and continuous aggregates. In the following example, you tier chunks that are more than three days old in the `example` hypertable.
```

File: use-timescale/data-tiering/creating-data-tiering-policy.md [hunk 2]
```diff
@@ -60,6 +60,32 @@ If you remove a tiering policy, the removal automatically prevents scheduled chu
 Any chunks that were already tiered won't be untiered automatically. You can use the [untier_chunk][untier-data] procedure 
 to untier chunks to local storage that have already been tiered.
 
+The procedure for adding and removing tiering policy for a continuous aggregate is identical to a hypertable. For the following example we'll use a continuous aggregate called `example_day_avg`.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/creating-data-tiering-policy.md]:
```suggestion
The procedure for adding and removing tiering policy for a continuous aggregate is identical to a hypertable. The following example uses a continuous aggregate called `example_day_avg`.
```

File: use-timescale/data-tiering/creating-data-tiering-policy.md [hunk 3]
```diff
@@ -60,6 +60,32 @@ If you remove a tiering policy, the removal automatically prevents scheduled chu
...
+
+<Procedure>
+
+### Adding a tiering policy for a continuous aggregate
+
+1. At the psql prompt, specify the continuous aggregate name and the interval, after which chunks need to be moved to the tiered storage:
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/creating-data-tiering-policy.md]:
```suggestion
1. At the psql prompt, specify the continuous aggregate name and the interval after which chunks are moved to  tiered storage:
```

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nice work, couple of small language suggestions.

**Comment by iroussos** [comment]
Thank you @billy-the-fish, suggestions look great! Applied all of them - back to you :-)

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**